libraries/RW/K8s/k8s_connection.py

```diff
@@ -106,42 +110,34 @@ def shell(
         if (
             (rsp.status != 200 or rsp.returncode > 0)
             and rsp.stderr != ""
-            and not any(partial_stderr in rsp.stderr for partial_stderr in K8sConnectionMixin.ALLOWED_STDERR)
+            and not any(partial_stderr in rsp.stderr for partial_stderr in K8sConnection.ALLOWED_STDERR)
         ):
             raise ValueError(
                 f"The shell service responded with HTTP: {rsp.status} RC: {rsp.returncode} and response: {rsp}"
             )
         logger.info("shell stdout: %s", rsp.stdout)
         return rsp.stdout
 
+    @staticmethod
     def template_workload(
-        self,
         workload_name: str,
         workload_namespace: str,
-        workload_container: str,
-        target_service: platform.Service=None,
-        kubeconfig: platform.Secret=None,
-        context: str="",
+        workload_container: str
     ) -> str:
         """Take in the workload variables and construct a valid string that specifies the namespace and container. 
 
         Args:
-            workload_name (str): a workload type in which a pod can be found such as deployment/my-deployment or statefulset/my-statefulset. Also accepts labels if starting with `-l`
+            workload_name (str): a workload type in which a pod can be found such as deployment/my-deployment or statefulset/my-statefulset
```

Comment:
I think there is some copy/paste overwrite here. Are you purposefully taking out the support to pass labels in the workload name?

Reply:
Gah - borked the git pull. Thank you for catching that.